src/model/index.ts

```diff
@@ -368,6 +383,20 @@ class Model implements WorkbookModel {
     return this.ratioCalculator.findRatio(value.id).ratio;
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public getConversionCommonStockClass(value: any): any {
```

Comment:
This function is responsible of finding the resulting common class for a warrant or option